detections/web/exploit_public_facing_application_via_apache_commons_text.yml

```diff
@@ -0,0 +1,79 @@
+search: '| tstats `security_content_summariesonly` count min(_time) as firstTime max(_time)
+  as lastTime from datamodel=Web where Web.http_method IN (POST, GET) by Web.src Web.status Web.uri_path Web.dest Web.http_method Web.uri_query Web.http_user_agent
+  | `drop_dm_object_name("Web")`
+  | eval utf=if(like(lower(uri_query),"%:utf-8:http%"),2,0)
+  | eval lookup = if(like(lower(uri_query), "%url%") OR like(lower(uri_query), "%dns%") OR like(lower(uri_query), "%script%"),2,0)
+  | eval other_lookups = if(like(lower(uri_query), "%env%") OR like(lower(uri_query), "%file%") OR like(lower(uri_query), "%java%") OR like(lower(uri_query), "%localhost%") OR like(lower(uri_query), "%properties%") OR like(lower(uri_query), "%resource%") OR like(lower(uri_query), "%sys%") OR like(lower(uri_query), "%xml%") OR like(lower(uri_query), "%base%"),1,0)
+  | addtotals fieldname=Score utf lookup other_lookups
+  | fields Score, src, dest, status, uri_query, uri_path, http_method, http_user_agent firstTime lastTime
+  | `security_content_ctime(firstTime)`
+  | `security_content_ctime(lastTime)`
+  | where Score >= 4
+  | `exploit_public_facing_application_via_apache_commons_text_filter`'
+how_to_implement: To implement, one must be collecting network traffic that is normalized in CIM and able to be queried via the Web datamodel. Or, take the chunks out needed and tie to a specific network source type to hunt in. Tune as needed, or remove the other_lookups statement. 
+known_false_positives: False positives are present when the values are set to 1 for utf and lookup. It's possible to raise this to TTP (direct notable) if removal of other_lookups occur and Score is raised to 2 (down from 4).
+references:
+  - https://sysdig.com/blog/cve-2022-42889-text4shell/
+  - https://nvd.nist.gov/vuln/detail/CVE-2022-42889
+  - https://lists.apache.org/thread/n2bd4vdsgkqh2tm14l1wyc3jyol7s1om
+  - https://www.rapid7.com/blog/post/2022/10/17/cve-2022-42889-keep-calm-and-stop-saying-4shell/
+tags:
+  analytic_story:
+  - Text4Shell CVE-2022-42889
+  cve:
+  - CVE-2022-42889
+  asset_type: Web Server
+  cis20:
+  - CIS 3
+  - CIS 5
+  - CIS 16
+  confidence: 70
+  context:
+  - Source:Application Log
+  - Stage:Execution
+  dataset:
+  - https://media.githubusercontent.com/media/splunk/attack_data/master/datasets/attack_techniques/T1190/text4shell/text4shell.log
+  impact: 70
+  kill_chain_phases:
+  - Exploitation
+  message: A URL was requested related to Text4Shell on $dest$ by $src$.
+  mitre_attack_id:
+  - T1505.003
+  - T1505
+  - T1190
+  nist:
+  - DE.CM
+  observable:
+  - name: dest
+    type: IP Address
+    role:
+    - Victim
+  - name: src
+    type: IP Address
+    role:
+    - Attacker
+  product:
+  - Splunk Enterprise
+  - Splunk Enterprise Security
+  - Splunk Cloud
+  required_fields:
```

Comment:
are all required fields listed here... seems like we have more fields in the search then listed here.